src/containers/forms/LogActivityForm.jsx

```diff
@@ -22,38 +22,38 @@ import labels from '../../fixtures/labels';
 import SNACKBARTIMEOUT from '../../constants/snackbarTimeout';
 
 /**
-   * @name LogActivityForm
-   * @summary Returns Form
-   * @returns Returns a form
-   */
+ * @name LogActivityForm
+ * @summary Returns Form
+ * @returns Returns a form
+ */
 class LogActivityForm extends Component {
-  static defaultProps = {
-    selectedItem: {},
-    updateSelectedItem: () => { },
-    createActivity: () => {},
-    updateActivity: () => {},
-    message: {
-      type: '',
-      text: '',
-    },
-  }
-  /**
-   * @name propTypes
-   * @type {PropType}
-   * @property {Array} categories - Array of activity categories
-  */
-  static propTypes = {
-    categories: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
-    closeModal: PropTypes.func.isRequired,
-    createActivity: PropTypes.func,
-    updateActivity: PropTypes.func,
-    selectedItem: PropTypes.shape({ id: PropTypes.string }),
-    updateSelectedItem: PropTypes.func,
-    message: PropTypes.shape({
-      type: PropTypes.string,
-      text: PropTypes.string,
-    }),
-  };
+	static defaultProps = {
```

Comment:
This file seems to be indented by 4 spaces instead of 2.

Reply:
🤷‍♂️ 
<img width="87" alt="screen shot 2018-09-13 at 15 57 00" src="https://user-images.githubusercontent.com/30657749/45489725-be7d4880-b76d-11e8-80ee-8f34386e4385.png">
Essentially I have not yet written a single line of code and my settings haveare set to two spaces.
So what's the way forward?